extensions/skins/conversation_v1/player.html

```diff
@@ -173,7 +173,7 @@
                       current-state-name="activeCard.stateName">
   </oppia-gadget-panel>
 
-  <div ng-if="explorationCompleted && isLoggedIn && !isInPreviewMode && !isIframed && currentProgressDotIndex === transcript.length - 1"
+  <div ng-if="isOnFinalCard() && isLoggedIn && !isInPreviewMode && !isIframed && currentProgressDotIndex === transcript.length - 1"
```

Comment:
There are other instances of explorationCompleted in this file that need to be handled, I think.